client/src/lib/session/hooks/useSubscribeToSession.ts

```diff
@@ -22,12 +20,7 @@ const useSubscribeToSession = (sessionId: LiveSession['id']) => {
           if (!snapshot.exists) {
             onSnapshot(undefined);
           }
-
-          const sessionState = getSessionState(
-            getData<SessionStateData>(snapshot),
-          );
-
-          onSnapshot(sessionState);
+          onSnapshot(SessionStateSchema.validateSync(snapshot.data()));
```

Comment:
Why do we need to validate on this end?

Reply:
Since this is directly from the database we get the transformation with this